src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -229,21 +228,16 @@ export default {
 				// extract query
 				const pageQuery = _get(args, 'route.query');
 
-				return Promise.all([
-					// query our targeted loan channel
-					client.query({
-						query: loanChannelQuery,
-						variables: _merge(
-							{
-								ids: [targetedLoanChannelID],
-								limit: loansPerPage
-							},
-							fromUrlParams(pageQuery)
-						)
-					}),
-					// experiment: add to basket interstitial
-					client.query({ query: experimentQuery, variables: { id: 'add_to_basket_v2' } }),
```

Comment:
move this query up to parent component so that the new experimental layout can use it as well.